go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/huandu/xstrings v1.3.0
 	github.com/json-iterator/go v1.1.9
+	github.com/mesos/mesos-go v0.0.10 // indirect
```

Comment:
Why this is added?

Reply:
I accidentally used apierrors from this repo, because of the Auto Import of VSCode, which I guess added it to the go.mod, but didn't remove it as I fixed the import. Sorry